client/components/homepage/Login.jsx

```diff
@@ -7,41 +7,67 @@ import socket from '../../clientUtilities/sockets';
 
 const Login = (props) => {
     return (
-      <div className="container-fluid">
-        <div className="col-md-8 col-md-offset-2">
-          <h1 className="text-center">Sign up!</h1>
-          <form className="col-md-12">
-            <input
-              className="col-md-12"
-              type="text"
-              placeholder="Enter a username"
-              onChange={(e) => { props.addUserName(e.target.value); }}
-            />
-            <input
-              className="col-md-12"
-              type="text"
-              placeholder="Enter an e-mail address"
-              onChange={(e) => { props.addUserEmail(e.target.value); }}
-            />
-            <button
-              onClick={() => {
-                socket.emit('userName submitted', {
-                  userName:props.userName,
-                  email: props.email,
-                  fbToken: null
-                });
-              }}
-            >
-              <Link to="/" onClick={(e) => { props.loggedIn(true); }}>Sign me up!</Link>
-            </button>
-            <Facebook history={props.history} />
-          </form>
-        </div>
+      <div>
+        <ul id="login">
+          <li>
+            <div className="row">
+              <div className="login-box col-md-4 col-md-offset-4">
+                <div className="social-buttons">
+                  <Facebook history={props.history} />
+                </div>
+                <center><p>or</p></center>
+                 <form className="form" role="form" method="post" action="login" acceptCharset="UTF-8" id="login-nav">
```

Comment:
I'm not sure why this keeps the page from refreshing like it used to when you hit enter on the sign in inputs, but I like it!